### ISSUE TITLE
Standalone programs for interactive documentation

### ISSUE THREAD

**Issue by viebel** [plasma-umass/browsix#44]
Hi there,

This project looks amazing!!!

I'd like to use it in order to create interactive documentation about the unix shell commands.

I was thinking of an integration of browsix into a tool of mine named [klipse](github.com/viebel/klipse). In order to do that all I need is a standalone javascript file that exports the shell commands and outputs to the console.

Then we could have a html page with several code snippets  that share the same environment and the reader would be able to play with the code and get a better understanding of the shell commands.

Does it make sense?
Do you think it's feasible?

**Comment by MGageMorgan** [comment]
Yes it makes sense, if I am reading this right: You want to create a web app that allows you to run command line snippets in the browser, and allow the user to modify them. Sounds like an educational project, and also a lot like the use cases they outlined for Browsix.

Is it feasible? Only if you can get Browsix working in the first place, which for some reason won't build for me. Props if you get this working. I'd love to see the result! XD
